crates/sui-protocol-config/src/lib.rs

```diff
@@ -1288,7 +1288,7 @@ impl ProtocolConfig {
             }
             17 => {
                 let mut cfg = Self::get_for_version_impl(version - 1, chain);
-                cfg.execution_version = Some(1);
+                cfg.feature_flags.upgraded_multisig_supported = true;
 
                 // Following flags are implied by this execution version.  Once support for earlier
                 // protocol versions is dropped, these flags can be removed:
```

Comment:
i think these comments need to follow `execution_version` or they will not make any sense